plugins/source/testing.go

```diff
@@ -8,14 +8,12 @@ import (
 	"github.com/cloudquery/plugin-sdk/specs"
 )
 
-type Validator func(t *testing.T, plugin *Plugin, resources []*schema.Resource)
-
 func TestPluginSync(t *testing.T, plugin *Plugin, spec specs.Source, opts ...TestPluginOption) {
 	t.Helper()
 
 	o := &testPluginOptions{
 		parallel:   true,
-		validators: []Validator{validatePlugin},
+		validators: []Validator{validateColumnsHaveData, validateTableNamePlural},
```

Comment:
`validateColumnsHaveData` make sense to have as a default but `validateTableNamePlural` seems like something to be defined by the plugin itself.